### DISCUSSION TITLE
Muted unexpectedly

### DISCUSSION THREAD

**Discussion by sonic3XE** [sonic3XE/evades-region-editor#11] [General]
i just posted an image in Channel#437060682693738497 and then got muted all the sudden. Briefly describe what did I do wrong. (Note: the image does not have NSFW content and Channel#437060682693738497 refers to #general-en)
Which rule number(s) did I break? Why did this happen?

**Comment by Ino5559** [comment]
you know me? you probably broke the rule, it’s a pity that I can’t make noise in the arena if I support that you didn’t do anything

**Reply by sonic3XE** [reply]
_i mean the rule number(s) which is edited from above_ oh wait
Refer to [the list of rules in Evades.io Discord Server](https://discord.com/channels/283154072675287040/759114335062327347/759121674598875156).

**Comment by Ino5559** [comment]
i not know what rule numbers ¯\_(ツ)_/¯


пт, 7 июн. 2024 г. в 00:02, sonic3XE ***@***.***>:
